src/library/validators.js

```diff
@@ -33,11 +32,11 @@ const checkPassword = str => {
   const score = computeScore(info)
 
   let force = null
-  if (score === Object.keys(info).length) {
-    force = 'high'
-  } else if (score >= 3 && (str.length >= 6)) {
-    force = 'medium'
-  } else force = 'low'
+  if (str.length < 6 || score < 2) {
```

Comment:
easier: 
```js
force = str.length < 6 ? 0 : score
```

Except if you have a case in mind where str.length > 6 while score is < 2 ?

Reply:
Yes if there are 6 digit password for example, the score will be 1 but the password will not be accepted as we expect at least 6 characters and 2 rules.